src/test/java/camp/nextstep/edu/kitchenpos/bo/OrderBoTest.java

```diff
@@ -0,0 +1,198 @@
+package camp.nextstep.edu.kitchenpos.bo;
+
+import camp.nextstep.edu.kitchenpos.dao.OrderDao;
+import camp.nextstep.edu.kitchenpos.dao.OrderLineItemDao;
+import camp.nextstep.edu.kitchenpos.dao.OrderTableDao;
+import camp.nextstep.edu.kitchenpos.dao.TableGroupDao;
+import camp.nextstep.edu.kitchenpos.model.Order;
+import camp.nextstep.edu.kitchenpos.model.OrderLineItem;
+import camp.nextstep.edu.kitchenpos.model.OrderTable;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+class OrderBoTest {
+
+  @Mock
+  private OrderDao orderDao;
+
+  @Mock
+  private OrderTableDao orderTableDao;
+
+  @Mock
+  private TableGroupDao tableGroupDao;
+
+  @Mock
+  private OrderLineItemDao orderLineItemDao;
+
+  @InjectMocks
+  private OrderBo orderBo;
+
+
+  @DisplayName("주문 목록을 볼 수 있다.")
+  @Test
+  public void list() throws Exception {
+    //given
+    List<Order> orders = Arrays.asList(createOrder(1L), createOrder(2L));
+    given(orderDao.findAll()).willReturn(orders);
+
+    //when
+    List<Order> actual = orderBo.list();
+
+    //then
+    assertThat(actual).isNotNull();
+    assertThat(actual).hasSize(orders.size());
+  }
+
+  @DisplayName("주문 상태를 변경할 수 있다.")
+  @Test
+  public void changeOrderStatus() throws Exception {
+    //given
+    Order order = createOrder(1L);
+    order.setOrderStatus("COOKING");
```

Comment:
```suggestion
    order.setOrderStatus(OrderStatus.COOKING.name());
```

작성되어 있는 enum을 활용하는 것은 어떨까요